azurerm_application_insights_web_test/main.tf

```diff
@@ -0,0 +1,39 @@
+# The module configure an web test from an url
+
+# Existing infrastructure
+data "azurerm_resource_group" "rg" {
+  name = "${local.azurerm_resource_group_name}"
+}
+
+data "azurerm_application_insights" "generic_web_tests" {
+  name                = "io-dev-ai-generic-web-tests"
```

Comment:
Why is this hardcoded?

Reply:
Because this is the application insight that has been created specifically for this purpose. And this tests are attached to it specifically.